### DISCUSSION TITLE
Table Creation Limits on Supabase [per DB]

### DISCUSSION THREAD

**Discussion by nthouliss** [supabase/supabase#5485] [Questions] [answered]
Hi!

I'd like to utilize the realtime subscriptions for my collaboration software, which allows users to edit a "board" in real time. I realise that as more connections are made to a table the longer the processing time becomes an idea I had to combat this was to create a database table per board, have users subscribe to that table, and then destroy the table once collaboration has finished. This would help with processing time and distribute load across multiple tables however I don't know what kind of limits I'd be up against and how many tables I can create on a single database in Supabase?

Please let me know if you need more clarification :)

**Comment by w3b6x9** [comment, marked as answer]
@nthouliss supabase doesn't limit the number of tables in a database and there's no current limit for realtime websocket connections. it'll depend on the hardware that your project runs on which is determined by your plan.

also, we are going to launch a multiplayer feature soon where users can send each other messages directly, thus bypassing the database. this will be more performant for collaborative apps such as yours.

**Reply by nthouliss** [reply]
Thanks! 

That new feature sounds awesome, is there an estimate to when this feature would be made available?

**Reply by w3b6x9** [reply]
@nthouliss we're aiming for early Q2 of this year!

**Reply by nthouliss** [reply]
Awesome! Appreciate it :)
